app/models/family.rb

```diff
@@ -278,7 +281,14 @@ class Family
   }
 
   scope :outstanding_verifications_expiring_on, lambda {|date|
-    where(:"eligibility_determination.outstanding_verification_earliest_due_date" => date.beginning_of_day)
+    where({
```

Comment:
Does this criteria include only subjects with evidence state for enrollment == true ?

Recommend making that a separate scope --outstanding_verifications_subjects_enrolled -- that can be chained here and into other scopes (and make it more obvious that non-enrolled family members are ignored)

Reply:
>Does this criteria include only subjects with evidence state for enrollment == true ?

Currently, no. Also not sure which attributes we should be checking on the `EvidenceState` or `EligibilityState` models - there is an `is_eligble` attribute, is that what you were referring to? 

There does not appear to be an `enrollment` attribute on either of those models.